### PR TITLE
Response Set Evaluation Modifications

### PR DESCRIPTION
**Pull request by ankush-bhatia**
I have made some changes to the ResponseSet Evaluation.

- Added new Method to evaluate API path.

Please let me know your thoughts on this. @johntmcintosh

### CODE REVIEW

File: Examples/SwiftExample/SwiftExample/AppDelegate.swift [hunk 1]
```diff
@@ -32,7 +32,8 @@ extension ResponseSet {
     static func makeTopRepositories() -> ResponseSet {
         let generator = FileBasedResponseGenerator(contentType: ContentType.applicationJson, directory: "LocalServer")
 
-        var set = ResponseSet(requestName: "search", evaluation: .suffix("search/repositories"))
+        //var set = ResponseSet(requestName: "search", evaluation: .suffix("search/repositories"))
+        var set = ResponseSet(requestName: "search", evaluation: .contains("search/repositories"))
```
Comment (johntmcintosh) [inline Examples/SwiftExample/SwiftExample/AppDelegate.swift]: This was helpful to see as a use-case, but I try not to leave code commented out in the code that ends up in master, so would you mind reverting this file to remove this change?

File: BarricadeKit/Core/ResponseSet/ResponseSetEvaluation.swift [hunk 2]
```diff
@@ -32,6 +32,12 @@ extension ResponseSetEvaluation {
         }
     }
 
+    public static func contains(_ string: String) -> ResponseSetEvaluation {
```
Comment (johntmcintosh) [inline BarricadeKit/Core/ResponseSet/ResponseSetEvaluation.swift:35]: This is great 👍 One request though... if you take a look at `ResponseSetEvaluationTests.swift` there should be a couple of test cases for testing success and failure of the existing `suffix` evaluation. Would you add similar test cases for this new evaluation?

**Review by johntmcintosh — CHANGES_REQUESTED** [review]
This is great, thanks for the PR! Two small requests for you, and then this'll be good to go.

File: Examples/SwiftExample/SwiftExample/AppDelegate.swift (hunk 1 above)
Comment (ankush-bhatia) [reply Examples/SwiftExample/SwiftExample/AppDelegate.swift]: Sure i will revert this change.

File: BarricadeKit/Core/ResponseSet/ResponseSetEvaluation.swift (hunk 2 above)
Comment (ankush-bhatia) [reply BarricadeKit/Core/ResponseSet/ResponseSetEvaluation.swift:35]: Added test cases for the new evaluations in the updated PR.

**Comment by johntmcintosh** [comment]
Thanks @ankush-bhatia !

**Pull request merged**